src/VisualStudio/IntegrationTest/TestSetup/TestTraceListener.cs

```diff
@@ -91,7 +91,14 @@ public override void WriteLine(string? message, string? category)
 
         private static void Exit(string? message)
         {
-            FatalError.ReportAndPropagate(new Exception(message));
+            if (message?.Contains("Pretty-listing introduced errors in error-free code") ?? false)
```

Comment:
Do we have a tracking bug for this? If the answer is "no" feel free just to file one to track why this was even added since that might require a bunch of archeology.

Reply:
➡️  Filed #70744